### PR TITLE
Add acceptance criteria to PR template

### PR DESCRIPTION
**Pull request by barmac**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,3 +9,24 @@ Please make sure you provide the relevant context.
...
+-->
+
+* [ ] Corresponds to the concept <!-- link document here -->
+* [ ] Corresponds to the design <!-- link document here -->
+
+__Definition of Done__
+
+* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
+* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
+* [ ] passes Continuos Integration checks
```
Comment (philippfromme) [inline .github/PULL_REQUEST_TEMPLATE.md:28]: Continuos ➡️ Continuous

Comment (barmac) [reply .github/PULL_REQUEST_TEMPLATE.md]: Lol, I used spelling suggested by the typo checker extension :D

**Comment by barmac** [comment]
> Cool! 👏
> 
> Will we roll this out to other projects as well? Zeebe Modeler, bpmn-js, ...

I think so, but first I want to have a template to copy from.

**Comment by oguzeroglu** [comment]
Looks good, however "Passes CI checks" step is sometimes blocked by codecov checks. This happens pretty often actually. What'd be our approach there?

**Comment by barmac** [comment]
I think it is similar to

> a single commit closes the issue via Closes #issuenr

In most of the cases it works like this, but sometimes we intentionally deviate from the standard, if it's reasonable.

**Comment by oguzeroglu** [comment]
Have we shipped the merge-me feature yet? Maybe @andreasgeier needs to approve this as well 😛

**Comment by oguzeroglu** [comment]
it works!!

**Pull request merged**